apps/src/lab2/lab2Redux.ts

```diff
@@ -16,7 +16,6 @@ import {
   LevelProperties,
   ProjectManagerStorageType,
   ProjectSources,
-  ProjectLevelData,
```

Comment:
Thanks for the clean up here and below!

Reply:
I'm surprised Drone picked up on them now, and not in some other PR!  cc @sanchitmalhotra126 in case you're curious.

Reply:
Interesting! Surprised that Drone didn't catch this, thanks for the cleanup!